File: src/main/java/no/navneet/SanntidsSykkelDataApplication.java
package no.navneet;

import io.dropwizard.Application;
import io.dropwizard.setup.Bootstrap;
import io.dropwizard.setup.Environment;

public class SanntidsSykkelDataApplication extends Application<SanntidsSykkelDataConfiguration> {

    public static void main(final String[] args) throws Exception {
        new SanntidsSykkelDataApplication().run(args);
    }

    @Override
    public String getName() {
        return "sanntidssykkeldata";
    }

    @Override
    public void initialize(final Bootstrap<SanntidsSykkelDataConfiguration> bootstrap) {
        // TODO: application initialization
    }

    @Override
    public void run(final SanntidsSykkelDataConfiguration configuration,
                    final Environment environment) {
        // TODO: implement application
    }

}

File: src/main/java/no/navneet/SanntidsSykkelDataApplication.kt
package no.navneet

import io.dropwizard.Application
import io.dropwizard.setup.Bootstrap
import io.dropwizard.setup.Environment
import java.lang.Exception

class SanntidsSykkelDataApplication : Application<SanntidsSykkelDataConfiguration?>() {
    override fun getName(): String {
        return "sanntidssykkeldata"
    }

    override fun initialize(bootstrap: Bootstrap<SanntidsSykkelDataConfiguration?>) {
        // TODO: application initialization
    }

    override fun run(
        configuration: SanntidsSykkelDataConfiguration?,
        environment: Environment
    ) {
        // TODO: implement application
    }

    companion object {
        @Throws(Exception::class)
        @JvmStatic
        fun main(args: Array<String>) {
            SanntidsSykkelDataApplication().run(*args)
        }
    }
}